vendor/jquery/component.dollar.js

```diff
@@ -0,0 +1,12 @@
+Ember.Component.reopen({
```

Comment:
I think its a good idea to always wrap things in vendor files with an IIFE:

```js
(function() {
  // contents here
})();
```

In this case it _probably_ doesn't matter, but if things grow over time we might leak globals (by declaring variables and such)...

Reply:
done